docs/index.md

```diff
@@ -74,113 +74,16 @@ The following items are limitations with the current release of the Red Hat Clou
 
 To use the Red Hat Cloud Services provider inside your Terraform configuration you must meet the following:
 
-* Completed [the ROSA getting started](https://console.redhat.com/openshift/create/rosa/getstarted) requirements
```

Comment:
Are you sure you ment to remove those lines?

Reply:
I'm changing the structure and flow of the documentation a bit. The registry will focus as a place for all the endpoints and helping developers, while the documentation side should be a deeper explanation and use. I'll keep revising the prereqs though.